plugins/services/diff/local.go

```diff
@@ -41,7 +41,8 @@ type config struct {
 	// respected for which is chosen. Each differ should return the same
 	// correct output, allowing any ordering to be used to prefer
 	// more optimimal implementations.
-	Order []string `toml:"default"`
+	Order  []string `toml:"default"`
+	SyncFs bool     `toml:"sync_fs"`
```

Comment:
There is already option in client side. I don't have strong opinion on that.
ping @containerd/committers @containerd/reviewers ~

Reply:
When it is set on the client, is it done for every call or only the last? Having it set via the toml config has its advantages if its something an administrator wants to enforce.

Reply:
Yeah. For the client side, it's done for one call. In plugin side, it can be done for every call.

Reply:
whether the contents of the following directory have changed when save image ?
```
[root@localhost io.containerd.snapshotter.v1.overlayfs]# ls snapshots/
1  3
```
if will not change we can  just do  one sync. @fuweid 

```
var cache sync.Map

func synFsOnce(file string) error {
	if _, ok := cache.Load(file); ok {
		return nil
	}
	result := doSyncFs(file)
	cache.Store(file, result)
	return result
}
```